src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/yflows/YFlowUpdateSpec.groovy

```diff
@@ -108,6 +111,92 @@ class YFlowUpdateSpec extends HealthCheckSpecification {
+
+        then: "Requested updates are reflected in the response and in 'get' API"
+        expect updateResponse, sameBeanAs(yFlow, ignores)
+        expect northboundV2.getYFlow(yFlow.YFlowId), sameBeanAs(yFlow, ignores)
+
+        and: "All related switches have no discrepancies"
+        involvedSwitches.each { sw ->
+            northbound.validateSwitch(sw.dpId).verifyRuleSectionsAreEmpty(["missing", "excess", "misconfigured"])
+            northbound.validateSwitch(sw.dpId).verifyMeterSectionsAreEmpty(["missing", "excess", "misconfigured"])
+        }
+
+        cleanup:
+        yFlow && yFlowHelper.deleteYFlow(yFlow.YFlowId)
+    }
+
+    @Tidy
+    def "User can partially update fields of one-switch y-flow"() {
+        given: "Existing one-switch y-flow"
+        def singleSwitch = topologyHelper.getRandomSwitch()
+        def switchId = singleSwitch.dpId
+        def yFlowRequest = yFlowHelper.singleSwitchYFlow(singleSwitch, false)
+        yFlowRequest.setMaxLatency(50)
+        yFlowRequest.setMaxLatencyTier2(100)
+        def yFlow = yFlowHelper.addYFlow(yFlowRequest)
+        def patch = YFlowPatchPayload.builder()
+                .maximumBandwidth(yFlow.maximumBandwidth * 2)
+        .pathComputationStrategy(PathComputationStrategy.MAX_LATENCY.toString().toLowerCase())
+        .encapsulationType(FlowEncapsulationType.TRANSIT_VLAN.toString().toLowerCase())
+        .maxLatency(yFlow.maxLatency * 2)
+        .maxLatencyTier2(yFlow.maxLatencyTier2 * 2)
+        .ignoreBandwidth(true)
+        .pinned(true)
+        .priority(10)
+        .strictBandwidth(false)
+        .description("updated description")
+        .build()
+        yFlow.setMaximumBandwidth(patch.getMaximumBandwidth())
+        yFlow.setPathComputationStrategy(patch.getPathComputationStrategy())
+        yFlow.setEncapsulationType(patch.getEncapsulationType())
+        yFlow.setMaxLatency(patch.getMaxLatency())
+        yFlow.setMaxLatencyTier2(patch.getMaxLatencyTier2())
+        yFlow.setIgnoreBandwidth(patch.getIgnoreBandwidth())
+        yFlow.setPinned(patch.getPinned())
+        yFlow.setPriority(patch.getPriority())
+        yFlow.setStrictBandwidth(patch.getStrictBandwidth())
+        yFlow.setDescription(patch.getDescription())
+
+
+        when: "Partial update the y-flow"
+        def updateResponse = yFlowHelper.partialUpdateYFlow(yFlow.YFlowId, patch)
+        def ignores = ["subFlows", "timeUpdate", "status"]
+
+        then: "Requested updates are reflected in the response and in 'get' API"
+        expect updateResponse, sameBeanAs(yFlow, ignores)
+        expect northboundV2.getYFlow(yFlow.YFlowId), sameBeanAs(yFlow, ignores)
+
+        and: "All related switches have no discrepancies"
+        northbound.validateSwitch(switchId).verifyRuleSectionsAreEmpty(["missing", "excess", "misconfigured"])
+        northbound.validateSwitch(switchId).verifyMeterSectionsAreEmpty(["missing", "excess", "misconfigured"])
```

Comment:
i would like to use v2 cause it already released (not blocker) - the same in the previous test

Reply:
Done.